### PR TITLE
Fix typos in english locales

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

File: client/app/locales/en.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
   "error": "Error",
   "error proper email": "Given email is not correct",
   "error email empty": "Given email is empty",
-  "Chose the language you want I use to speak with you:": "Choose the language you want to see:",
+  "Choose the language you want I use to speak with you:": "Choose the language you want to see:",
```
Comment (nono) [inline client/app/locales/en.json]: @clochix you have updated the key, not the wording. I think it's still 'Chose' in code. Can you change it too?

Comment (frankrousseau) [inline client/app/locales/en.json]: Please follow the guidelines when it's about changing the localization key. That one is wrong.

**Comment by frankrousseau** [comment]
ping @clochix

**Comment by frankrousseau** [comment]
I did the changes.

**Pull request merged**